rules/visual-design-specifications/rule.md

```diff
@@ -19,87 +19,90 @@ created: 2009-02-28T09:45:02.000Z
 
-Often it's best to start with a low-fidelity mockup to get across a concept or rough storyboard. Then complete a high-fidelity mockup to communicate the look and feel, and if time permits create an interactive prototype.
+### 3 Elements of a storyboard
 
-### Low-fidelity Mockups
+There are 3 common storyboard elements: a user story, visuals, and corresponding captions.
 
-Creating a low-fidelity design mockup can be achieved through various methods including wireframes, paper prototypes, and other similar techniques. It can be valuable to have the product owner on call when creating a low-fidelity mockup to get immediate feedback and direction.
+**User story:** Storyboards are based on a specific scenario. The role that corresponds to that scenario must be clearly specified. A short text description of the scenario or story must clear enough that a team member or stakeholder could understand what is depicted before looking at the visuals.
 
-Start by identifying the key features and functionality that the product should have. This will help you focus your efforts and ensure that your mockup accurately represents your intended product.
+Also See: [Do you use User Stories format when appropriate?](https://www.ssw.com.au/rules/spec-do-you-use-user-stories/)
 
-::: img-large
-![Figure: Example of a hand-drawn mockup. Nice and quick for early concept design](hand-drawn-mockup.jpg)
-:::
+**Visuals:** The scenario is visually depicted in a sequential manner using sketches or photos. These images can range from quick, low-fidelity drawings to detailed, high-fidelity artifacts, depending on the purpose and intended audience of the storyboard. The visuals include relevant details such as the user’s environment or a sketch of the screen being interacted with.
 
-Then create a basic wireframe of your design. A wireframe is a visual representation of the layout and structure of your website or app. You can create digital wireframes using tools like Figma, Adobe XD, or Sketch. Alternatively, you can use paper and pencil to sketch out a rough wireframe by hand. Add details to your wireframe by adding boxes and labels for different features and elements (e.g. buttons, input fields, and images). 
+Also See: [Do you know the value of User Journey Mapping?](https://www.ssw.com.au/rules/user-journey-mapping/)
+
+**Captions:** Each visual should include corresponding s to describes the user’s actions, environment, device, and so on.
 
 ::: good
-![Figure: Good example – Example of wireframes (created in Figma)](figma_wireframe_app_screenshot.png)
+![Figure: Good example – a simple quick hand-drawn storyboards specific scenario with all 3  elements.](basic-storyboard.png)
 :::
 
-Once your low-fidelity design mockup has been approved, you can use it as a reference point for creating a high-fidelity mockup. 
+::: bad
+![Figure: Bad example – UML is virtually impossible to get clients to understand software specifications](bad-uml.jpg)
```

Comment:
I think this is unrelated and out of context

Reply:
How about I change the caption to:

Figure: A UML is good to analyze, design, and document systems but virtually impossible to get clients to understand software specifications

Reply:
I don't like UML being part of the content because I find it unrelated...

Reply:
It was on Adam's version wo show a bad example trying to use UML to document spec or show to the client.

Personally I also think it is a good "bad example" to show it here.